sites/intellij-guide/contents/tips/find-usages/index.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2020-10-05
+title: 'Find Usages'
+technologies: []
+topics: [navigation]
+author: hs
+subtitle: 'Use Find Usages to search for references in your code base.'
+thumbnail: ./thumbnail.png
+cardThumbnail: ./card.png
+shortVideo:
+  poster: ./tip.png
+  url: https://www.youtube.com/watch?v=bcqK8awmwyE
```

Comment:
12 sec is perfect length. I do worry that the width/height is going to be unreadable on smaller devices on Twitter. I've been doing my tips at 960x540, with much bigger IDE font (18pt) and editor font (20pt).

Reply:
That's good to know. For now, I'm converting the gifs we have into MP4 to minimise the time it takes to get something live and looking good for IntelliJ's birthday next year. Ie, reusing what we have - fortunately I think they're all around this length. I uploaded this one on my YouTube but I think they probably should be on the coroporate one. I'll check that.